object-search/app.py

```diff
@@ -4,44 +4,42 @@
 import os
 import shutil
 import sys
+from glob import glob
 import requests
 
 import click
 from jina.flow import Flow
 from jina import Document
 from jina.clients.sugary_io import _input_lines
 from jina.logging import default_logger as logger
+from jina.logging.profile import TimeContext
 
-
-num_docs = os.environ.get('MAX_DOCS', 16)
-data_path = 'data/**/*.jpg'
-batch_size = 16
-
-
-def clean_workdir():
-    if os.path.exists(os.environ['WORKDIR']):
-        shutil.rmtree(os.environ['WORKDIR'])
-        logger.warning('Workspace deleted')
+MAX_DOCS = int(os.environ.get("JINA_MAX_DOCS", 50))
+IMAGE_SRC = 'data/f8k/images/*.jpg'
 
 
 def config():
-    parallel = 1 if sys.argv[1] == 'index' else 1
-    shards = 1
-
-    os.environ['PARALLEL'] = str(parallel)
-    os.environ['SHARDS'] = str(shards)
+    os.environ["JINA_DATA_FILE"] = os.environ.get("JINA_DATA_FILE", IMAGE_SRC)
+    os.environ['PARALLEL'] = str(1)
```

Comment:
I guess we dont need the str conversion here anymore? same for below. I know it was there before e.g. for the port, but it looks kinda weird?

```suggestion
    os.environ['PARALLEL'] = '1'
```

Reply:
Thanks, https://github.com/jina-ai/examples/pull/477/commits/563e81154ab69492cab4d5db3d6d24de77922b0d